man/fi_mr.3.md

```diff
@@ -681,6 +698,22 @@ FI_HMEM_ZE interfaces.  The driver and device index values represent
 their 0-based positions in arrays returned from zeDriverGet and zeDeviceGet,
 respectively.
 
+## av
+
+For memory registration being allocated against a domain configured with
+FI_AV_AUTH_KEY, av is used to define the fid_av which contains the authorization
+keys to be associated with the memory region. If the domain is also opened with
+FI_MR_ENDPOINT, the specified AV must be the same AV bound to the endpoint.
+
+By default, the memory region will be associated with all authorization keys
+in the AV.
+
+## key
+
+If the domain was opened with FI_DIRECTED_RECV, key can be used to limit the
```

Comment:
I tried to re-used the FI_DIRECTED_RECV for RMA/AMOs. For clarity, should a new macro be defined for directed one-sided operations?

Reply:
We need a wider opinion on this.  At first glance, this seems reasonable to me.